test/extended/olm/olmv1.go

```diff
@@ -142,28 +147,82 @@ var _ = g.Describe("[sig-olmv1] OLMv1 operator installation", func() {
 		o.Expect(err).NotTo(o.HaveOccurred())
 
 		g.By("waiting for the ClusterExtention to be installed")
-		err = wait.PollUntilContextTimeout(ctx, time.Second, 5*time.Minute, true, func(ctx context.Context) (done bool, err error) {
-			var conditions []metav1.Condition
-			output, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("clusterextensions.olm.operatorframework.io", "install-test-ce", "-o=jsonpath={.status.conditions}").Output()
-			if err != nil {
-				return false, err
-			}
-			// no data yet, so try again
-			if output == "" {
-				return false, nil
-			}
-			err = json.Unmarshal([]byte(output), &conditions)
-			if err != nil {
-				return false, fmt.Errorf("error in json.Unmarshal(%v): %v", output, err)
-			}
-			if !meta.IsStatusConditionPresentAndEqual(conditions, "Progressing", metav1.ConditionFalse) {
-				return false, nil
-			}
-			if !meta.IsStatusConditionPresentAndEqual(conditions, "Installed", metav1.ConditionTrue) {
-				return false, nil
-			}
-			return true, nil
-		})
+		err = wait.PollUntilContextTimeout(ctx, time.Second, 5*time.Minute, true, func(ctx context.Context) (bool, error) { return ApplyClusterExtensionAndWait(oc, "install-test-ce") })
```

Comment:
Can the `return` statemment and following `})` be put on their own lines for readability?